src/components/PieCharts/CustomActiveDonut/CustomActiveDonutView.js

```diff
@@ -116,12 +118,18 @@ class CustomActiveDonut extends PureComponent {
     const {
       data: DataObj, textColor, colors, paddingSpace,
       titleLocation, titleAlignment, titleText,
-      fontSize, fontWeight, fontFamily,
+      fontSize, fontWeight, fontFamily, showTotalCount,
```

Comment:
CustomActiveDonut.defaultProps = {
  titleText: 'Cases',
  paddingSpace: 0,
};
Can you pass a default prop

Reply:
Done ✅  Added.